src/storage/src/hummock/compactor/fast_compactor_runner.rs

```diff
@@ -189,14 +193,17 @@ impl ConcatSstableIterator {
         sst_infos: Vec<SstableInfo>,
         sstable_store: SstableStoreRef,
         task_progress: Arc<TaskProgress>,
+        compactor_metrics: Arc<CompactorMetrics>,
     ) -> Self {
+        compactor_metrics.compaction_reading_sst_counts.inc();
```

Comment:
What are we counting for this metric? The number of SSTs being read or the number levels being read? If we inc on concat iter creation and drop on concat iter destruction, it is the latter one.